pkg/bff/models/v1/organization.go

```diff
@@ -204,7 +204,7 @@ func NewRegisterForm() *RegistrationForm {
 // response to ensure that we're able to submit valid forms and that validation only
 // occurs in one place in the code.
 func (r *RegistrationForm) ReadyToSubmit(network string) bool {
-	if r.VaspCategories == nil || r.Entity == nil || r.Contacts == nil || r.Trixo == nil {
+	if r.Entity == nil || r.Contacts == nil || r.Trixo == nil {
```

Comment:
I did review the GDS registration validation form and it looks like vasp categories aren't validated there either, so this should be fine even when sent to GDS.

https://github.com/trisacrypto/trisa/blob/main/pkg/trisa/gds/models/v1beta1/models.go#L91